templates/project_root_containers/CMakeLists.txt

```diff
@@ -96,7 +96,21 @@ if(SIMPATH)
 endif()
 
 find_package2(PUBLIC ROOT  VERSION 6.10.00  REQUIRED)
```

Comment:
Let's either drop the version requirement or update it.

Reply:
Dropped for now.

I think, the version requirement here should reflect which version of ROOT the project_*_containers need themselves.